scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+# aws-cli
+# RUN sudo curl -O https://bootstrap.pypa.io/get-pip.py \
+# 	&& python3 get-pip.py --user \
+# 	&& pip3 install awscli --upgrade --user \
+# 	&& sudo rm get-pip.py
+RUN pip install awscli --upgrade --user
+RUN sudo ln -s /home/qa/.local/bin/aws /usr/local/bin/aws
+
+########################### TEST-8 #############################
+
+# sbt build
+RUN whoami
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)